applications/nrf_desktop/src/modules/fast_pair.c

```diff
@@ -10,11 +10,14 @@
 #define MODULE fast_pair_app
 #include <caf/events/module_state_event.h>
 #include <caf/events/ble_common_event.h>
+#include "ble_dongle_peer_event.h"
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_FAST_PAIR_LOG_LEVEL);
 
-static uint8_t local_id = BT_ID_DEFAULT;
+static uint8_t selected_stack_id = BT_ID_DEFAULT;
+static uint8_t dongle_app_id;
+static bool fp_provider_enabled;
```

Comment:
`static bool fp_provider_enabled = true;` -> The provider is enabled by default unless dongle peer is selected.